v2/index.html

```diff
@@ -1160,32 +1160,36 @@ <h4>Chords</h4>
         </section>
         <section>
             <h4>Beaming</h4>
+            <div class="issue" data-number="24"></div>
             <div class="issue" data-number="45"></div>
-            <table class="simple">
-                <thead>
-                <tr>
-                    <th>Code</th>
-                    <th>Remarks</th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr>
-                    <td><code>{</code></td>
-                    <td>beginning of beaming</td>
-                </tr>
-                <tr>
-                    <td><code>}</code></td>
-                    <td>end of beaming</td>
-                </tr>
-                </tbody>
-            </table>
-
+            <div class="issue" data-number="84"></div>
+            <p>
+                Beamed notes are encoded using braces <code>{}</code>.
+            </p>
+            <p>
+                Beam groups MUST start with an opening brace <code>{</code>, and
+                MUST end with a closing brace, <code>}</code>. Nested beam groups
+                MUST NOT occur.
```

Comment:
Except in a grace group `'8{CDqq{GF}rEF}`
<img width="168" alt="image" src="https://github.com/rism-digital/pae-code-spec/assets/689412/14d093b0-652a-4505-b2a9-5a8c80e406c2">

Reply:
Right